StripeFinancialConnections/StripeFinancialConnections/Source/Native/PartnerAuth/PartnerAuthViewController.swift

```diff
@@ -409,6 +409,15 @@ final class PartnerAuthViewController: UIViewController {
 
     private func openInstitutionAuthenticationWebView(authSession: FinancialConnectionsAuthSession) {
         guard let urlString = authSession.url, let url = URL(string: urlString) else {
+            dataSource
+                .analyticsClient
+                .logUnexpectedError(
+                    FinancialConnectionsSheetError.unknown(
+                        debugDescription: "Invalid or NULL URL returned from auth session"
+                    ),
+                    errorName: "InvalidAuthSessionURL",
+                    pane: .partnerAuth
+                )
             assertionFailure("Expected to get a URL back from authorization session.")
             return
```

Comment:
Is this valid still? should we do smth else? maybe reload the auth session or smth?

Reply:
@vardges-stripe 

OK I added `navigateBack()` so it would just pop the user back to institution picker

it's a simple solution that would help to reload the auth session; IMO we shouldn't overcomplicate something that should ideally never happen

https://user-images.githubusercontent.com/105514761/225468971-d19104ac-2177-433d-af83-b94522958566.mp4